src/Box/index.js

```diff
@@ -27,14 +38,12 @@ const Box = React.forwardRef((props, ref) => (
 ));
 
 Box.propTypes = {
-  ...COMMON.propTypes,
-  ...LAYOUT.propTypes,
-  ...POSITION.propTypes,
-  ...FLEXBOX.propTypes,
-  ...BORDERS.propTypes,
+  ...propTypes.styleProps,
   theme: PropTypes.object,
 };
 
 Box.displayName = 'Box';
 
+export { StyledBox };
```

Comment:
What's the difference between a named export of `StyledBox` and the default Box export?

Reply:
StyledBox is a styled component. Box is a react component that returns a StyledBox instance.